Makefile

```diff
@@ -2,7 +2,7 @@
 .PHONY: build test check clean fmt cdc kafka_consumer coverage \
 	integration_test_build integration_test integration_test_mysql integration_test_kafka bank \
 	dm dm-master dm-worker dmctl dm-syncer dm_coverage \
-	engine tiflow df-master-client df-demo df-chaos-case
+	engine tiflow df-demo df-chaos-case
```

Comment:
Can we rename the `df-demo df-chaos-case` to `tiflow-demo tiflow-chaos-case`?

Reply:
Yes, they will be renamed in another PR, in case of this PR too large.